mlflow/pipelines/steps/evaluate.py

```diff
@@ -197,12 +198,17 @@ def my_warn(*args, **kwargs):
                             "explainability_algorithm": "kernel",
                             "explainability_nsamples": 10,
                             "pos_label": self.positive_class,
+                            "metric_prefix": "validation_",
```

Comment:
I recall in Databricks AutoML, this prefix is `val_`. Shall we converge the prefix here?

Reply:
Thanks for the catch! Yes, AutoML uses `val_`. I'll make the change.